### PR TITLE
Try to solve the problem that the boss can not be killed and does not die together

### PR DESCRIPTION
**Pull request by w5860363**
Try to solve https://github.com/TrinityCore/TrinityCore/issues/27562

But due to another bug https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/scripts/Outland/BlackTemple/boss_illidari_council.cpp# L554 BOSS is constantly regaining health

Not yet completed

The spell will still reset creature health indefinitely

### CODE REVIEW

**Comment by Keader** [comment]
It looks a hack to me.
If i remember right, they should stay in "live" and die with SPELL_QUIET_SUICIDE
If they're not dying with it, need check why.

File: src/server/scripts/Outland/BlackTemple/boss_illidari_council.cpp [hunk 1]
```diff
@@ -268,8 +268,18 @@ struct IllidariCouncilBossAI : public BossAI
...
-            damage = me->GetHealth() - 1;
+        if (who == me)
+            return;
+
+        for (uint32 bossData : CouncilData)
+        {
+            if (Creature* council = instance->GetCreature(bossData))
+                if (council != me && damage < council->GetHealth())
+                {
+                    council->ModifyHealth(-int32(damage));
```
Comment (Keader) [inline src/server/scripts/Outland/BlackTemple/boss_illidari_council.cpp:279]: hack, they should die for SPELL_QUIET_SUICIDE

**Review by Keader — CHANGES_REQUESTED** [review]
hack 
please check why they're not dying for SPELL_QUIET_SUICIDE

**Comment by w5860363** [comment]
@Keader This is just an attempt. There are many problems at present

Fighting can't be balanced. Their health value is constantly restored to full, so I think this is the reason why they can't die of suicide

**Comment by Keader** [comment]
Some core change caused it, since it was working when was scripted (and script dont change)
check why they're restoring life and why quiet suicide is not killing too.

**Pull request closed without merging**